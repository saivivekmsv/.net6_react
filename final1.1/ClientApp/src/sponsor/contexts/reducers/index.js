export * from "./app-layout";
export * from "./manage-company";
export * from "./create-plan";
export * from "./manage-eligibility";
export * from "./manage-census";
export * from "./manage-payroll";
export * from "./manage-maintenance";
export * from "./manage-report";
export * from "./manage-mapper";
