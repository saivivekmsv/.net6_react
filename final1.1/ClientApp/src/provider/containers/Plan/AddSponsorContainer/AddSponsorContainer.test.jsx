import React from "react";
import { fireEvent, render } from "@testing-library/react";
import { BrowserRouter as Router, Route } from "react-router-dom";
import AddSponsorContainer from "./AddSponsorContainer";
import { CreatePlanProvider } from "../../../contexts";

describe("AddSponsorContainer", () => {
  const props = {};
  test("to match snapshot active", () => {
    const { baseElement } = render(
      <CreatePlanProvider>
        <Router>
          <AddSponsorContainer {...props} />
        </Router>
      </CreatePlanProvider>
    );
    expect(baseElement).toMatchSnapshot();
  });

  test("to match on form submit", () => {
    const wrapper = render(
      <CreatePlanProvider>
        <Router>
          <AddSponsorContainer {...props} />
        </Router>
      </CreatePlanProvider>
    );
    const form = wrapper.container.querySelector("form");
    fireEvent.submit(form);
    expect(wrapper.baseElement).toMatchSnapshot();
  });
});
