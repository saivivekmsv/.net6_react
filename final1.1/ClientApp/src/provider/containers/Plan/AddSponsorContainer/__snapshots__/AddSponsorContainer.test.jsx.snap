// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddSponsorContainer to match on form submit 1`] = `
<body>
  <div>
    <form
      class="h-100"
    >
      <div
        class="d-flex flex-column manage-company h-100"
      >
        <div
          class="d-flex layout-header align-items-center"
        >
          <div>
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/home"
                    >
                      Home
                    </a>
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/plans"
                    >
                      Manage Plan
                    </a>
                  </a>
                </li>
              </ol>
            </nav>
            <h5 />
          </div>
          <div
            class="d-flex align-items-center justify-content-center tab-wrapper"
          />
          <div
            class="ml-auto"
          >
            <div>
              <a
                href="/plans"
              >
                <button
                  class="btn btn-secondary"
                  flow="add,save"
                  link="/plans"
                  type="button"
                >
                  Cancel
                </button>
              </a>
                 
              <button
                class="btn btn-primary"
                flow="add"
                type="submit"
              >
                Save
              </button>
                 
            </div>
          </div>
        </div>
        <div
          class="layout-content h-100 overflow-auto"
        >
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  First Name
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Last Name
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  SSN
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-4"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  E Mail
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  name="email"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Phone No.
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  name="phoneNumber"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;

exports[`AddSponsorContainer to match snapshot active 1`] = `
<body>
  <div>
    <form
      class="h-100"
    >
      <div
        class="d-flex flex-column manage-company h-100"
      >
        <div
          class="d-flex layout-header align-items-center"
        >
          <div>
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/home"
                    >
                      Home
                    </a>
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/plans"
                    >
                      Manage Plan
                    </a>
                  </a>
                </li>
              </ol>
            </nav>
            <h5 />
          </div>
          <div
            class="d-flex align-items-center justify-content-center tab-wrapper"
          />
          <div
            class="ml-auto"
          >
            <div>
              <a
                href="/plans"
              >
                <button
                  class="btn btn-secondary"
                  flow="add,save"
                  link="/plans"
                  type="button"
                >
                  Cancel
                </button>
              </a>
                 
              <button
                class="btn btn-primary"
                flow="add"
                type="submit"
              >
                Save
              </button>
                 
            </div>
          </div>
        </div>
        <div
          class="layout-content h-100 overflow-auto"
        >
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  First Name
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Last Name
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  SSN
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-4"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  E Mail
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  name="email"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-md-2"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Phone No.
                   
                </label>
                <input
                  autocomplete="off"
                  class="w-100 form-control"
                  name="phoneNumber"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
